ocs_ci/helpers/helpers.py

```diff
@@ -5882,3 +5882,113 @@ def create_rbd_deviceclass_storageclass(
     sc_obj = create_resource(**sc_data)
     sc_obj.reload()
     return sc_obj
+
+
+def find_cephblockpoolradosnamespace(storageclient_uid=None):
+    """
+    Find the cephblockpoolradosnamespace related to a storageclient. This should run on provider cluster in a
+        provider mode setup.
+
+    Args:
+        storageclient_id(string): The uid of the storageclient for which the cephblockpoolradosnamespace to be fetched
+
+    Returns:
+        str: The name of the cephblockpoolradosnamespace, if present
+
+    """
+    if not storageclient_uid:
```

Comment:
You can run the code also with RunWithProviderConfigContextIfAvailable context (https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/__init__.py#L488) to make sure that it is executed on provider.

Reply:
Ack. Also, considering the case where we need to have more than one provider in the run, for now the context will be switched from the test case.